src/cascadia/TerminalApp/Tab.idl

```diff
@@ -1,18 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT license.
-import "ShortcutActionDispatch.idl";
-
-namespace TerminalApp
-{
-    [default_interface] runtimeclass Tab : Windows.UI.Xaml.Data.INotifyPropertyChanged
-    {
-        String Title { get; };
-        Windows.UI.Xaml.Controls.IconSource IconSource { get; };

```

Comment:
So just as a sanity check, `IconSource` wasn't actually being used anywhere, was it?

Reply:
On account of it compiles and runs fine without IconSource, I am inclined to say yes.